modules/rpc/cookie.js

```diff
@@ -94,13 +94,14 @@ function mkDir(dirPath, root) {
 ** RPC cookie is regenerated at every particld startup
 */
 function getAuth(options) {
+
   if (options.rpcuser && options.rpcpassword) {
     return options.rpcuser + ':' + options.rpcpassword;
   }
 
-  // const COOKIE_FILE = findCookiePath() + `${options.testnet ? '/testnet' : ''}/.cookie`;
-  const COOKIE_FILE = findCookiePath() + (options.testnet ? '/testnet' : '') + '/.cookie';
   let auth;
+  let testnet = options.testnet ? '/testnet' : ''
```

Comment:
Is an additional variable necessary here?

Reply:
just for avoiding 80+ lines, should I remove it ?

Reply:
changed the previous comment